include/powsybl/iidm/converter/ExportOptions.hpp

```diff
@@ -255,8 +293,12 @@ class ExportOptions {
     std::map<std::string, std::string> m_extensionsVersions;
 
     std::string m_version;
+
+    IidmVersionIncompatibilityBehavior m_iidmVersionIncompatibilityBehavior = IidmVersionIncompatibilityBehavior::THROW_EXCEPTION;
 };
 
+std::ostream& operator<<(std::ostream& stream, const ExportOptions::IidmVersionIncompatibilityBehavior& value);
```

Comment:
I think it's not needed: there is a template function that should work well if the `getNames` method has been specialized

Reply:
https://github.com/powsybl/powsybl-iidm4cpp/issues/223